benchmark/single-source/SetTests.swift

```diff
@@ -49,11 +49,7 @@ let setY = Set(half ..< size)         // 200 ..< 400
 // Two sets with 100% overlap, but different bucket counts (let's not make it
 // too easy...)
 let setP = Set(0 ..< size)
-let setQ: Set<Int> = {
-  var set = Set(0 ..< size)
-  set.reserveCapacity(2 * size)
-  return set
-}()
+let setQ = Set(0 ..< size)
```

Comment:
The whole point of having sets `P` and `Q` with identical contents but different capacity is described in the above comment.

The `setQ` should not be changed, please revert this part back to as it was. My previous [review comments](https://github.com/apple/swift/pull/24156#discussion_r280842909) were specific to `arrayQ` only.